ShopperInsights/src/main/java/com/braintreepayments/api/ShopperInsightsClient.kt

```diff
@@ -43,6 +42,14 @@ class ShopperInsightsClient @VisibleForTesting internal constructor(
     ) {
         braintreeClient.sendAnalyticsEvent(GET_RECOMMENDED_PAYMENTS_STARTED)
 
+        if (isTokenizationKey()) {
+            callbackFailure(
+                callback = callback,
+                error = BraintreeException("Invalid authorization. This feature can only be used with a client token.")
```

Comment:
Do we have in our documentation that tokenization token is not allowed with ShopperInsights module?

Reply:
Yep, in the `note` section here: https://developer.paypal.com/braintree/docs/guides/shopper-insights/ios/v6#integration